PyART/models/seob.py

```diff
@@ -0,0 +1,139 @@
+import os, subprocess
+import numpy as np
+try:
+    import pyseobnr.generate_waveform as SEOB
+except ModuleNotFoundError:
+    print("WARNING: pyseobnr not installed.")
+import lal
+
+from ..waveform import Waveform
+from ..utils import wf_utils as wfu
+
+class Waveform_SEOB(Waveform):
+    """
+    Class for SEOBNRv5HM waveforms
+    Uses pySEOBNR package
+    """
+    def __init__(
+                    self,
+                    pars=None,
+                ):
+        super().__init__()
+        self.pars  = pars
```

Comment:
What happens if `pars` is not specified? You probably want to fail gracefully in that case, or handle the exception setting some"default" parameters.

Reply:
Added check and RuntimeError if no pars specified.